doc/netplan.md

```diff
@@ -967,6 +967,10 @@ more general information about tunnels.
 
 :   Defines the address of the remote endpoint of the tunnel.
 
+``ttl`` (scalar)
```

Comment:
Could you please append ` – since **0.100**` to this line, so we can document that this setting will be introduced with the current development version (0.100)